magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -62,7 +62,7 @@ object CloudFormation extends DeploymentType {
   ).default("AMI")
 
   override def perAppActions = {
-    case "updateStack" => pkg => (lookup, parameters, stack) => {
+    case "updateStack" => pkg => (logger, lookup, parameters, stack) => {
```

Comment:
I really thought I'd found all of these... apparently my late night find/replace skillz are lacking.